sql/core/src/test/resources/sql-tests/results/linear-regression.sql.out

```diff
@@ -1,7 +1,7 @@
 -- Automatically generated by SQLQueryTestSuite
 -- !query
 CREATE OR REPLACE TEMPORARY VIEW testRegression AS SELECT * FROM VALUES
-(1, 10, null), (2, 10, 11), (2, 20, 22), (2, 25, null), (2, 30, 35)
+(1, 10, null), (2, 10, 11), (2, 20, 22), (2, 25, null), (2, 30, 35), (2, null, 40)
```

Comment:
A tuple is added with the value of `y` is null, it should be filtered out during calculation, so the output related to `RegrSlope` & `RegrIntercept` in the output remains unchanged.